zoo-deployment.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Zookeeper should be represented as a StatefulSet, since its stores metadata for coordinating and other things